code/game/objects/items.dm

```diff
@@ -581,6 +581,13 @@
 
 		//END MONKEY
 
+/obj/item/reveal_blood()
+	if(was_bloodied && !fluorescent)
+		fluorescent = 1
+		blood_color = COLOR_LUMINOL
+		blood_overlay.color = COLOR_LUMINOL

```

Comment:
Ignore me if this is incorrect, but wouldn't this cause **all** items of the same type with the same blood colour turn into the luminol colour?